cfgov/v1/tests/models/test_base.py

```diff
@@ -7,6 +7,7 @@
 from wagtail import blocks
 from wagtail.models import Page, Site
 
+from ask_cfpb.models import PortalSearchPage
```

Comment:
I think we want to avoid having v1 code depend on ask_cfpb code if possible. Is there some way to move this test to the ask_cfpb project? It seems like all that's being reused is the make_page logic which is pretty minimal.